2019-oscon/mloncode.md

```diff
@@ -0,0 +1,24 @@
+# ML on Code: Machine Learning Will Change Programming
+
+**Updates**:  https://github.com/src-d/conferences/issues/103
+
+**Sent on**:  TBA
+
+**Status**:   on preparation
+
+**Author**:   Francesc Campoy
+
+**Slides**:   TBA
+
+**Proposal**: TBA
+
+**Abstract**:
```

Comment:
Given that you have InnerSource in the title, I would develop the abstract to focus on the cross-team collaboration, duplicate code and silos detection.